src/pages/Authentication.tsx

```diff
@@ -0,0 +1,114 @@
+import { useState, FC } from "react";
+import formImage from "../images/ask_form.jpg";
+import logo from "../images/logo.svg";
+
+import {
+	Button,
+	VStack,
+	Flex,
+	Text,
+	Stack,
+	Image,
+	ButtonGroup,
+} from "@chakra-ui/react";
+
+import { GoMarkGithub } from "react-icons/go";
+import LoginFrom from "../components/loginForm";
+import RegisterForm from "../components/registerForm";
+
+const Authentication: FC = () => {
+	const [currentForm, setCurrentForm] = useState("sign in");
```

Comment:
You can use `enum` as the `currentForm` value here or at set the `currentForm` type to be either 'sign in ' or 'sign up' using the code below:
```suggestion
	const [currentForm, setCurrentForm] = useState<"sign in" | "sign up">("sign in");
```

Reply:
Updated, please review it.